test/e2e/nstemplatetier_test.go

```diff
@@ -41,7 +41,7 @@ func TestNSTemplateTiers(t *testing.T) {
 	testingtiers := CreateAndApproveSignup(t, hostAwait, testingTiersName)
 
 	// all tiers to check - keep the basic as the last one, it will verify downgrade back to the default tier at the end of the test
-	tiersToCheck := []string{"advanced", "team", "basicdeactivationdisabled", "basic"}
+	tiersToCheck := []string{"advanced", "team", "basicdeactivationdisabled", "basic", "test"}
```

Comment:
could you leave `basic` as the last tier, as the comment above explains?

Reply:
Who reads the comments? ;)

Good catch.

Reply:
Fixed in https://github.com/codeready-toolchain/toolchain-e2e/pull/236/commits/3f1b5b2a8f04bd84da388d9c9bf68f71e6576d96

Reply:
thanks!